examples/batch/argo-workflows-batch/README.md

```diff
@@ -0,0 +1,239 @@
+    [[1, 2, 3, 4]]
+    [[1, 2, 3, 4]]
+    [[1, 2, 3, 4]]
+    [[1, 2, 3, 4]]
+    [[1, 2, 3, 4]]
+    [[1, 2, 3, 4]]
+    [[1, 2, 3, 4]]
+    [[1, 2, 3, 4]]
+
+
+### Upload the file to our minio
+
+
+```python
+!mc mb minio-seldon/data
+!mc cp assets/input-data.txt minio-seldon/data/
+```
+
+    [m[32;1mBucket created successfully `minio-seldon/data`.[0m
+    ...-data.txt:  146.48 KiB / 146.48 KiB ┃▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓┃ 2.03 MiB/s 0s[0m[0m[m[32;1m
+
+### Create Argo Workflow
+
+In order to create our argo workflow we have made it simple so you can leverage the power of the helm charts.
+
+Before we dive into the contents of the full helm chart, let's first give it a try with some of the settings.
+
+We will run a batch job that will set up a Seldon Deployment with 10 replicas and 100 batch client workers to send requests.
+
+
+```python
+!helm template seldon-batch-workflow helm-charts/seldon-batch-workflow/ \
+    --set workflow.name=seldon-batch-process \
+    --set seldonDeployment.name=sklearn \
+    --set seldonDeployment.replicas=10 \
+    --set batchWorker.workers=100 \
+    --set batchWorker.payloadType=ndarray \
+    --set batchWorker.dataType=data \
+    | argo submit -
+```
+
+    Name:                seldon-batch-process
+    Namespace:           default
+    ServiceAccount:      default
+    Status:              Pending
+    Created:             Mon Jun 08 11:57:56 +0100 (1 second ago)
+
+
+
+```python
+!argo list
+```
+
+    NAME                   STATUS    AGE   DURATION   PRIORITY
+    seldon-batch-process   Running   3s    3s         0
+
+
+
+```python
+!argo get seldon-batch-process
+```
+
+    Name:                seldon-batch-process
+    Namespace:           default
+    ServiceAccount:      default
+    Status:              Succeeded
+    Created:             Mon Jun 08 11:57:56 +0100 (1 minute ago)
+    Started:             Mon Jun 08 11:57:57 +0100 (1 minute ago)
+    Finished:            Mon Jun 08 11:59:33 +0100 (10 seconds ago)
+    Duration:            1 minute 36 seconds
+    
+    [39mSTEP[0m                                                             PODNAME                          DURATION  MESSAGE
+     [32m✔[0m seldon-batch-process (seldon-batch-process)                                                              
+     ├---[32m✔[0m create-seldon-resource (create-seldon-resource-template)  seldon-batch-process-3626514072  1s        
+     ├---[32m✔[0m wait-seldon-resource (wait-seldon-resource-template)      seldon-batch-process-2052519094  32s       
+     ├---[32m✔[0m download-object-store (download-object-store-template)    seldon-batch-process-1257652469  3s        
+     ├---[32m✔[0m process-batch-inputs (process-batch-inputs-template)      seldon-batch-process-2033515954  50s       
+     └---[32m✔[0m upload-object-store (upload-object-store-template)        seldon-batch-process-2123074048  3s        
+
+
+
+```python
+!argo logs -w seldon-batch-process 
+```
```

Comment:
`-w` is not an option in my argo so this line fails.
```
argo version
argo: v2.8.1
  BuildDate: 2020-05-28T23:39:46Z
  GitCommit: 0fff4b21c21c5ff5adbb5ff62c68e67edd95d6b8
  GitTreeState: clean
  GitTag: v2.8.1
  GoVersion: go1.13.4
  Compiler: gc
  Platform: linux/amd64

```

Reply:
Ok cool, I've added a command that attempts both so it supports multiple versions, as 2.6 is relatively recent